crates/weaver_emit/src/attributes.rs

```diff
@@ -577,6 +584,7 @@ mod tests {
                         note: None,
                         stability: None,
                         deprecated: None,
+                        annotations: None,
```

Comment:
@jerbly / @lquerel After seeing this type of change bloat PRs a thousand times,

How do we feel about using the DEFAULT syntax so this isn't so tedious when safely adding new fields?

e.g.:

```rust
EnumEntriesSpec {
  id: "non-default".to_owned(),
  ...Default::default(),
}
```

Instead of these more verbose updates where it's harder to see the content of what has changed...

Reply:
I agree